packages/cosmwasm/src/lcdapi/wasm.spec.ts

```diff
@@ -335,7 +335,7 @@ describe("wasm", () => {
       // upload data
       const hackatom = getHackatom();
       const result = await uploadContract(client, wallet, hackatom);
-      expect(result.code).toBeFalsy();
+      assert(!result.code);
```

Comment:
why is this better?

we are checking for `0` or `undefined` if I understand

Reply:
Because `assert` throws if the condition is falsy, `expect` reports an error an continues. But in case the transaction failed, all the following checks are pointless and just hide the problem.